packages/@react-spectrum/list/src/listview.css

```diff
@@ -191,6 +186,50 @@
     padding-inline-start: var(--spectrum-global-dimension-size-75);
   }
 
+    &.react-spectrum-ListViewItem-dragPreview {
+      border: 1px solid var(--spectrum-dropzone-border-color-selected-hover);
+      border-radius: var(--spectrum-alias-border-radius-regular);
+      background-color: var(--spectrum-table-background-color, var(--spectrum-global-color-gray-50));
+
+      .react-spectrum-ListViewItem-grid {
+        grid-template-areas:
+          "icon image content     . badge"
+          "icon image description . badge"
+        ;
+        grid-template-columns: auto auto auto 1fr;
+      }
+
+      .react-spectrum-ListViewItem-badge {
```

Comment:
if you had an item with a description, I imagine that badge will be aligned to the top or possibly stretched, can you verify?